### PR TITLE
Removes the unintelligible obnoxious accents 

### PR DESCRIPTION
**Pull request by Wirpy**
## About The Pull Request

Removes grenzlehoft and dwarf accents 

## Why It's Good For The Game

The dwarf accent is unintelligible and borderline obnoxious. For a lot of the words you'd never figure out what it means unless you keep the list open on a second monitor or something (or if you play dwarf). It's just a poor implementation of a accent that frankly doesn't really make a lot of sense and just worsens the experience of engaging with someone playing a dwarf. If a dwarf wants to be unintelligible they already have their own language. 

The grenzlehoft accent is barely an accent at all, the list is short and some of the words like Kaiser isn't even fitting since it's refering to a emperor rather than a king. Just also poorly implementated, an accent isnt just replace a few words with full ones from another language. 

TL:DR Less obnoxious and unintelligible interactions, rejoice for we can now understand dwarfs again.

### CODE REVIEW

**Comment by Gyrandola** [comment]
Dwarf isnt really hard just learn english

**Comment by Wirpy** [comment]
> Dwarf isnt really hard just learn english

drack dracks drackin athwart yill naur yont abeich kis scratcher bowlt ablo wysed jyner cheetie gart gar stramash jarg tassie sned 

Just a few, the list goes on long enough. Some words aren't bad and one can figure em out / they're still recognizle enough but a large part are just nonsense creating a unintelligible obnoxious experience. 
Like what's the point? 

It's straight up just gibberish.

**Comment by Zebraman** [comment]
Based PR, speed merge this, dwarf accent is absolute cancer

**Comment by TheManWithNoHands** [comment]
The accents are fun, but honestly just poor qualitly. Its fun to hear the dwarf accent, but for actually communicating its ass. Like, you cant even understand other dwarfs as dwarf. What kinda bs is that?

**Comment by YosemiteYam** [comment]
The dwarf accent is cancer and this comes from someone who plays dwarves, I'm tired of having to spell out words like T H I S because the "accent" literally invents gibberish words. 

The Grenzelhoft shit is also stupid as its literally just words being replaced by german words instead of an actual accent, if I have to hear "Mein Keiser" one more time I'm gonna rage

**Comment by ChungusGamer666** [comment]
I already told you in discord that the accents are not gonna be removed. Open a dwarf accent nerf and it will be merged.
